beets/autotag/mb.py

```diff
@@ -276,11 +276,18 @@ def album_info(release):
         disambig.append(release.get('disambiguation'))
     info.albumdisambig = u', '.join(disambig)
 
-    # Release type not always populated.
-    if 'type' in release['release-group']:
-        reltype = release['release-group']['type']
-        if reltype:
-            info.albumtype = reltype.lower()
+    # Considers all release types (both primary and secondary) and stores as a comma-separated string
+    all_types = []
+    if 'primary-type' in release['release-group']:
+        rel_primarytype = release['release-group']['primary-type']
+        if rel_primarytype:
```

Comment:
Hey !
Yes this was intentional
line 285 confirms if the key exists or not. The value for the key 'secondary-type-list' will always be a list, so if the key exists, it can either be an empty or full of elements, and in either way, the for loop handles the situation.